backend/src/main/java/dev/projects/backend/controller/PersonController.java

```diff
@@ -0,0 +1,121 @@
+package dev.projects.backend.controller;
+
+import dev.projects.backend.collection.Person;
+import dev.projects.backend.dto.PersonDTO;
+import dev.projects.backend.enums.LoginRole;
+import dev.projects.backend.service.PersonService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
every endpoint is starting with 'person' u should use it up here at the global level